paddle/fluid/pybind/inference_api.cc

```diff
@@ -249,13 +257,41 @@ void BindAnalysisConfig(py::module *m) {
       .def("cpu_math_library_num_threads",
            &AnalysisConfig::cpu_math_library_num_threads)
       .def("to_native_config", &AnalysisConfig::ToNativeConfig)
+#ifdef PADDLE_WITH_MKLDNN
+      .def("enable_quantizer", &AnalysisConfig::EnableMkldnnQuantizer)
+      .def("quantizer_enabled", &AnalysisConfig::mkldnn_quantizer_enabled)
```

Comment:
Do you need polish `quantizer_enabled`?
- it is similar to `enable_quantizer`
- I don't see its usage in this PR.